src/pages/RestrictedAction/Workspace/WorkspaceUserRestrictedAction.tsx

```diff
@@ -28,7 +27,7 @@ function WorkspaceUserRestrictedAction({policyID}: WorkspaceUserRestrictedAction
 
     const openPolicyExpenseReport = useCallback(() => {
         const reportID = ReportUtils.findPolicyExpenseChatByPolicyID(policyID)?.reportID ?? '-1';
-        Report.openReport(reportID);
+        // Navigation.resetToHome();
```

Comment:
Got it. I think the following may solve our problem. Can you also please test this out?
```suggestion
        Navigation.closeRHPFlow();
```

https://github.com/Expensify/App/assets/3004659/c94db1c8-60cc-40bf-bf90-6e7cf9b0cc54